usr/share/rear/verify/DP/default/400_verify_dp.sh

```diff
@@ -8,9 +9,9 @@ StopIfError "Data Protector Cell Manager not set in /etc/opt/omni/client/cell_se
 
 if test "$PING" ; then
 	ping -c 1 "${CELL_SERVER}" >/dev/null 2>&1
```

Comment:
Same as https://github.com/rear/rear/pull/2443/files#r447487016
I wonder if STDOUT and STDERR redirection to /dev/null is needed here?
Cf. "What to do with stdin, stdout, and stderr" in
https://github.com/rear/rear/wiki/Coding-Style